apps/linear_algebra/src/blas_l2_generators.cpp

```diff
@@ -155,21 +157,18 @@ class GEMVGenerator :
             A_.dim(0).set_min(0).dim(1).set_min(0);
             x_.dim(0).set_bounds(0, A_.height());
             y_.dim(0).set_bounds(0, A_.width());
-            result.output_buffer().dim(0).set_bounds(0, A_.width());
         }
 
-        Func output("output");
-        output(i) = result(i);
+        // TODO: is this necessary? Doesn't result already have the necessary specializations? 
```

Comment:
Looks like it's a workaround from: 

https://github.com/halide/Halide/commit/cf999bf71939261bdcbb92d87fc4d07db5770732

We probably have the tools to deal with this properly now. I'd add a TODO: delete this pointless memcpy